physicalplan/model/operator.go

```diff
@@ -13,4 +13,6 @@ type VectorOperator interface {
 	Next(ctx context.Context) ([]StepVector, error)
 	Series(ctx context.Context) ([]labels.Labels, error)
 	GetPool() *VectorPool
+
+	Explain() (me string, next []VectorOperator)
```

Comment:
Why do we need to return `next []VectorOperator`?

Reply:
For concurrent operators - it shows sharding in amazing way